MAUI/DataForm/migration.md

```diff
@@ -628,6 +687,81 @@ The following properties of the `Xamarin SfDataForm` is moved to the [DataFormDe
 </tr>
 </table>
 
+#### TextInputLayoutSettings
+
+<table>
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[ContainerType](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.TextInputLayoutSettings.html#Syncfusion_XForms_DataForm_TextInputLayoutSettings_ContainerType)'| markdownify }}</td>
+<td>ContainerType</td>
+<td>Gets or sets a value that determines the container types for floating label layout.</td>
```

Comment:
<td>Gets or sets a value determining the container types for floating label layout.</td>